Source/Core/AudioCommon/AudioCommon.h

```diff
@@ -16,6 +16,9 @@ class Mixer;
 
```

Comment:
```suggestion
#include <mutex>

```
Add to headers in order to maintain compatibility with other platforms.